source/actions/actions.ts

```diff
@@ -250,7 +251,9 @@ export const setSituationBranch = (id: number): Action => ({
 	id,
 })
 
-const setSimulation = (simulation: Simulation): Action => ({
+const setSimulation = (
+	simulation: Pick<Simulation, 'config' | 'url'>
+): AnyAction => ({
```

Comment:
Pourquoi repasser à un type plus générique `AnyAction` ici alors que l'on a déjà le type `Action` ?

Reply:
Itération dont j'ai oublié la raison, je n'arrive pas à reproduire l'erreur de linter en repassant sur `Action`, je revert